### ISSUE TITLE
Tests and CI

### ISSUE THREAD

**Issue by PurHur** [dutchiexl/BehatHtmlFormatterPlugin#115]
The old behat tests for the behat formatter arent working anymore. The test tested the old HTML layout very hardcoded. Should we proceed like this? 
Since we now can load different templates we could create a own simplified template just for the testing pipeline. 

What do you think? @donkidd

**Comment by donkidd** [comment]
I think that is simple tests for that are fine, I was also thinking that we should probably have some tests out there to test the actual classes doing the work.  So the behat tests seem like more end to end type tests that make sure that the entire plugin works.  But should we also have some unit tests to test & validate the actual classes that are behind the entire plugin?

**Comment by PurHur** [comment]
Yes that should be tested too since i feel like there are some basic bugs in the basic Feature/Scenario classes. But the behat test would be a simple "it just works now" test.
